keras_cv/models/__init__.py

```diff
@@ -60,3 +60,13 @@
 from keras_cv.models.resnet_v2 import ResNet152V2
 from keras_cv.models.stable_diffusion import StableDiffusion
 from keras_cv.models.vgg19 import VGG19
+from keras_cv.models.vit import ViT_B_16
```

Comment:
can we rename these to remove the `_`s?   ViT_B_16 deviates from the rest of the Keras naming convention.  I guess We can do ViTB16 instead?

Reply:
Deal! Updating in a moment :)